realm/realm-library/src/testObjectServer/kotlin/io/realm/ObfuscatorHelper.kt

```diff
@@ -53,9 +53,9 @@ object ObfuscatorHelper {
 
     val CUSTOM_FUNCTION_ORIGINAL_INPUT = """
 {
-  "mail":"myfakemail@mongodb.com",
+  "mail":"${TestHelper.getRandomEmail()}",
```

Comment:
This won't work. This file is used for JVM unit tests and the test helper is located inside the Android instrumentation test package, so it won't find it.

Reply:
That's why the build is failing: https://ci.realm.io/blue/organizations/jenkins/realm%2Frealm-java/detail/PR-6993/4/pipeline/74